localization/english/culture/wc_culture_eras_l_english.yml

```
@@ -0,0 +1,5 @@
+﻿l_english:
+
+ #ERAS
+ culture_era_early_modern:0 "Early Modern"
+ culture_era_early_modern_desc:0 "The Early Modern Era marks the beggining of a new era. Old ways are forgotten as a new age of enlightenment dawns on the people of Azeroth."
```

Made a few minor changes.
```suggestion
 culture_era_early_modern_desc:0 "The Early Modern Era marks the beginning of a new era. The old ways have been forgotten as a new age of enlightenment dawns on the people of Azeroth."
```